### ISSUE TITLE
Text Editor Feedback

### ISSUE THREAD

**Issue by pgensler** [jupyterlab/jupyterlab#3919]
It would be nice to see what syntax highlighting I have in a given text file, similar to Atom/VS Code in the bottom right of the file.

It seems like when using an R script with dark theme, the cursor is missing?
![image](https://user-images.githubusercontent.com/7308115/36462143-39d9a484-1687-11e8-83f0-09dc5a62bedb.png)

**Comment by ian-r-rose** [comment]
The cursor issue has been fixed by #3962 and should be in the next release.

As for showing the theme: the text editor has been rather bare-bones for a while, but there are certainly a few things that would be nice to show. I wonder if we should buckle down and make a toolbar for the editor.

**Comment by saulshanabrook** [comment]
This might be a good thing to put in the status bar? https://github.com/jupyterlab/jupyterlab-statusbar

**Comment by pgensler** [comment]
@saulshanabrook yes, I think thats the proper place. It might be worthwhile to see if you can leverage parts of VS Code into the text editor? Not sure how card that could be to integrate, but something to think about.

**Comment by saulshanabrook** [comment]
Part of that is being done in the Monaco editor integration, which is the editor in VS Code: https://github.com/jupyterlab/jupyterlab/issues/135 Yeah it would be great!
